packages/grafana-schema/src/schema/dashboard/v2alpha0/dashboard.gen.ts

```diff
@@ -700,6 +700,7 @@ export interface PanelSpec {
 	links: DataLink[];
 	data: QueryGroupKind;
 	vizConfig: VizConfigKind;
+	displayMode?: "default" | "transparent";
```

Comment:
didn't we agree we just keep it as bool?

Reply:
Yes :) but I looked at [VizPanelState](https://github.com/grafana/scenes/blob/d78ce21e95cbefe57a759c80e0b935ac7e1e1462/packages/scenes/src/components/VizPanel/VizPanel.tsx#L51) and displayMode is defined like this so I decided to keep it consistent with that